README.md

```diff
@@ -29,6 +29,11 @@ Lagent is a lightweight open-source framework that allows users to efficiently b
     <img src="https://img.shields.io/badge/Python-007ACC?style=for-the-badge&logo=python&logoColor=yellow" alt="python" />
   </a>
 
+  ### All Thanks To Our Contributors:
```

Comment:
It might be better to put it to another position, e.g., to the place just above citation.

Reply:
Fixed the changes ! @ZwwWayne